integration-libs/cdc/manage-users/manage-users.service.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Move this service file to `manage-users-list`

Reply:
done